crates/re_data_store/src/store_db.rs

```diff
@@ -477,3 +491,57 @@ impl StoreDb {
             .map(|info| (info.application_id.0.as_str(), info.started))
     }
 }
+
+// ----------------------------------------------------------------------------
+
+pub struct IngestionStatistics {
+    e2e_latency_sec_history: Mutex<emath::History<f32>>,
```

Comment:
Low stakes, but an RwLock would make more sense I think

Reply:
it's only ever using the write-lock though, because it flushes old stats on-read

Reply:
ah, missed that!